### PR TITLE
352: Point-awardable conditions

### PR DESCRIPTION
**Pull request by pimotte**
これを書いておくのはしました。＝ "I did the thing of writing this to prepare for the future"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -416,6 +416,24 @@ One turn consists of proposing one rule-change and having it voted on.
...
+
+A rule implementing a point-awardable condition must at least specifiy:
+
+- The condition under which points may be awarded
+- How many points may be awarded
+
+The number of points for a point-awardable condition will be awarded upon merging
+of the PR that fulfills the condition, if the PR reflects this awarding of points.
+
+Players cannot obtain points using a point-awardable condition introduced that PR.
```
Comment (MrHug) [inline README.md]: introduced in _that_ PR.

File: README.md [hunk 2]
```diff
@@ -416,6 +416,24 @@ One turn consists of proposing one rule-change and having it voted on.
...
+
+Upon request of another player, a the initiator of a PR must state how many points their PR adds
```
Comment (MrHug) [inline README.md]: ~~a~~ the initiator

**Comment by pimotte** [comment]
Fixed

**Comment by MrHug** [comment]
:+1:

**Comment by pimotte** [comment]
As per 349, I'm issuing a warning.

**Comment by pimotte** [comment]
72 hours have passed, so this is 2/2. Merging.

**Pull request merged**